src/components/orgs/UserManagement.vue

```diff
@@ -269,40 +282,54 @@ export default {
     async onSubmit() {
       this.loadingAddingUser = true;
 
-      try {
-        const email = this.userSearch.toLowerCase();
+      const email = this.userSearch.toLowerCase();
 
-        if (this.users.find((user) => user.email === email)) {
-          this.userError = this.alreadyAddedText;
-          return false;
-        }
+      if (!/^\w+([-+.']\w+)*@\w+([-.]\w+)*\.\w+([-.]\w+)*$/.test(email)) {
```

Comment:
Create an util to help with masks

Reply:
I created temporally a object named rules, for future we think more about how we could centralize those rules.